tests/docs/test_docs.py

```diff
@@ -12,24 +12,28 @@ def path_to_docs(self):
     def test_html(self):
         wd = os.getcwd()
         os.chdir(os.path.sep.join(self.path_to_docs))
-
-        response = subprocess.run(["make", "html-noplot"])
-        self.assertTrue(response.returncode == 0)
-        # response = subprocess.call(["make", "html"], shell=True)  # Needed for local test on Windows
-        # self.assertTrue(response == 0)
+        
+        try:
+            response = subprocess.run(["make", "html-noplot"])
+            self.assertTrue(response.returncode == 0)
+        except:
+            response = subprocess.call(["make", "html"], shell=True)  # Needed for local test on Windows
+            self.assertTrue(response == 0)
```

Comment:
is this actually needed on windows still? it's been a while. Also this seems odd, because it basically runs the test twice if it fails... You probably need to catch a specific Error.

Reply:
Yes, It appears to be necessary. And the test failed right away when trying to build the docs so I'm not overly concerned about the timing right now.

Reply:
If I recall correctly, we used to use Travis which ran on Linux. But now we are using Azure which is a Microsoft product. I think the entire set of tests makes sure that everything works on any OS, but that means the additional testing of things for Windows OS. Make sense?

Reply:
Please change it to a specific exception then, or replace it with an `if platform.system() == "Windows` block then. See https://docs.python.org/3/library/platform.html#platform.system. I do not like having the catch all except block here.

Reply:
This works for me

Reply:
Just pointing this out, but your most recent change actually stops the doctests from running on the test suite.

Reply:
This is now fixed